EntAppProject/src/main/java/com/jobook/EntAppProject/Services/IJobService.java

```diff
@@ -8,15 +8,17 @@
 public interface IJobService {
 	/**
 	 * Get all jobs that related to userId provided by looking at the employee and customer Id
-	 * @param userId
-	 * @return
+	 * @param userId the user id to get all the jobs for

```

Comment:
Your JavaDocs need more info, otherwise they are not very useful when they get pulled up in your IDE. Notice how I added the exception, and more detail in general